video-library/src/components/Trending/TrendingVideos/JOURNALS.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import "./JOURNALS.css";
+function JOURNALS() {
+  return (
+    <div>
+      <div className="journals-row">
+        <iframe
+          className="thumbnail"
+          width="560"
+          height="315"
+          src="https://www.youtube.com/embed/VQUH8VLE0YE"
```

Comment:
DRY: This could be called from constant and a component can be created which can help in creating many video card from data